code/src/main/java/com/onlinecontacttracing/storage/data-storage/PositiveUserWithMessage.java

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 Google LLC
+//
```

Comment:
I'm concerned why we have this license here. We should be extremely careful about this.